src/main/java/org/opentripplanner/routing/api/request/RoutingRequest.java

```diff
@@ -759,12 +760,24 @@
      */
     public DataOverlayParameters dataOverlay = null;
 
+    /**
+     * Raptor can print all events when arriving at stops to stderrr. For developers only.
+     */
+    public Set<StopLocation> debugRaptorStops;
 
     /**
-     * Raptor can print all events when arriving at stops to system error. For developers only.
+     * Raptor can print all events for a path to stderr. For developers only.
+     *
+     * @see #debugRaptorPathFromStopIndex
      */
-    public DebugRaptor raptorDebuging = null;
```

Comment:
I would prefer to keep the nested request object, this class is too big and should be broken up in smaller pieces. Also, the RaptorDebug options are not part of the public api, so it is nice to keep the "noice" in this class at a minimum.